frame/system/benchmarking/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+// Benchmarks for Utility Pallet
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::Encode;
+use sp_std::vec;
+use sp_std::prelude::*;
+use sp_core::{ChangesTrieConfiguration, storage::well_known_keys};
+use sp_runtime::traits::Hash;
+use frame_benchmarking::{benchmarks, account};
+use frame_support::storage::{self, StorageMap};
+use frame_system::{Module as System, Call, RawOrigin, DigestItemOf, AccountInfo};
+
+mod mock;
+
+const SEED: u32 = 0;
+
+pub struct Module<T: Trait>(System<T>);
+pub trait Trait: frame_system::Trait {}
+
+benchmarks! {
+	_ { }
+
+	remark {
+		// # of Bytes
+		let b in 0 .. 16_384;
```

Comment:
Why that number?
Btw. do we want to limit the length of the remark in any way or are the byte fees enough disincentive?

Reply:
yeah, byte fees alone here should be good. It does not add any computational overhead afaik. This number has shown up in other places where we test a range of byte lengths, so I have mostly mimicked it here. It should be enough for us to see a clear trend.

Actual wasm size is around 300K - 400K if I understand correctly. I could update the value here if you think that is better